components/support/ktx/src/main/java/mozilla/components/support/ktx/android/view/View.kt

```diff
@@ -105,6 +106,22 @@ fun View.toScope(): CoroutineScope {
     return scope
 }
 
+/**
+ * Finds and returns the first [View] in the hierarchy matching the given predicate.
```

Comment:
nit: Finds the first a view in the hierarchy, for which the provided predicate is true.